lib/wechat/corp_api.rb

```diff
@@ -24,6 +24,15 @@ def agent(agentid)
       get 'agent/get', params: { agentid: agentid }
     end
 
+    def checkin(useridlist, starttime = Time.now.beginning_of_day, endtime = Time.now.end_of_day, opencheckindatatype = 3)
+      post 'checkin/getcheckindata', JSON.generate(
+        useridlist: useridlist,
```

Comment:
Still better put `useridlist` at end of hash, maybe not relevant, but still better just following the document sequence.

Reply:
Normally we need to fetch checkin data of the current day, thus 
`starttime` and `endtime` will be mostly default as the beginning of the current day and the end of the current day. `opencheckindatatype` will be mostly equal to `3`, which means fetching the whole list. The only changing argument at most time is `users
useridlist`